docs/logging.rst

```diff
@@ -15,3 +15,19 @@ messages.
     logger = logging.getLogger('django_auth_ldap')
     logger.addHandler(logging.StreamHandler())
     logger.setLevel(logging.DEBUG)
+
+Or if you're using Django's `LOGGING setting`_, you can add an entry to your
+logging config:
+
+.. _LOGGING setting: https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-LOGGING
```

Comment:
The docs are configured to generate intersphinx links. You can use that instead of hardcoding links to Django docs directly.

Reply:
I can't get the intersphinx reference working at all. Any clues on how to get this link working with intersphinx?

Reply:
I believe something like the following would work:

```
Or if you're using Django's :setting:`LOGGING` setting, you can add an entry to
your logging config.
```

Or:

```
Or if you're using Django's :setting:`LOGGING setting <LOGGING>`, you can add
an entry to your logging config.
```